storage/statedb/cache_redis.go

```diff
@@ -101,16 +105,33 @@ func (cache *RedisCache) Has(k []byte) ([]byte, bool) {
 	return val, true
 }
 
-func (cache *RedisCache) Publish(channel string, msg string) error {
+func (cache *RedisCache) publish(channel string, msg string) error {
 	return cache.client.Publish(channel, msg).Err()
 }
 
-func (cache *RedisCache) Subscribe(channel string) *redis.PubSub {
-	return cache.client.Subscribe(channel)
+func (cache *RedisCache) subscribe(channel string) *redis.PubSub {
```

Comment:
```suggestion
// subscribe subscribes to a redis stream
// It will return an existing subscription if any.
func (cache *RedisCache) subscribe(channel string) *redis.PubSub {
```
Maybe a comment for when `cache.pubSub` is previously set?

Reply:
Added comments. Thanks!